ONSdigital/eq-runner-maintenance-page
# Introduce Nginx and Github Actions

### What is the context of this PR?
- introduce an Nginx web server and Docker build to return the `service.html` page for any requests served by the container
- introduce a Github Action PR workflow to build the image and push to Dockerhub and GAR

### How to review 
- follow the README ensuring it makes sense and that the steps detailed build a functioning Nginx container
- ensure the container returns the appropriate responses for requests
- ensure the PR image is built and pushed to Dockerhub and GAR
